packages/react/package.json

```diff
@@ -61,7 +61,6 @@
     "@testing-library/react": "11.2.0",
     "@testing-library/react-hooks": "^8.0.1",
     "@testing-library/user-event": "^12.8.1",
-    "@types/cleave.js": "1.4.4",
     "@types/jest": "^26.0.1",
     "@types/jest-axe": "^3.5.3",
     "@types/react": "17.0.2",
```

Comment:
It seems that you can remove this line as well and the root package.jsons resolutions apply and everything works! `"@types/react": "17.0.2",
`

Reply:
`@types/react-dom` can also be removed 🤔

Reply:
I'm a bit skeptic about removing both of them from React package's devDependencies. After the removal those won't be listed anywhere else than "resolutions". So they are not listed as any kind of project dependencies anymore. 

If those types are removed, @types/react-dom is not installed in any node_modules. But there are no errors in my VSCode.

Maybe the best fix could be removing those types? 

Docs that say they are needed:

https://react.dev/learn/typescript
https://blog.logrocket.com/how-to-use-typescript-react-tutorial-examples/

Google generally says they are not needed until typescript complains about mismatching types.

So which is it 😄

Reply:
Shoot, I made the comment on the wrong line! I meant you can remove `"@types/react": "17.0.2",` from resolutions, not devDependencies, and everything works for me at least. Sorry!

Reply:
I would imagine that those two are needed in devDependencies though...

Reply:
Well, it would have worked 😄

Reply:
I removed the "@types/react": "17.0.2" from React and got no errors. Thx!

Reply:
Ok! This is good for me. I think it's all well and good that the packages are in React packages dependencies 👍